src/lib/style/bootstrap4_backports/_base-import.scss

```diff
@@ -70,7 +71,14 @@
     }
 
     .btn-light-primary-#{$color-variant} {
-      @include button-variant(#dfdfdf, #dfdfdf, $color, $color);
+      @include button-variant(
+        #dfdfdf,
+        #dfdfdf,
+        $color,
+        $color,
+        color.scale($color, $lightness: -20%),
+        color.scale($color, $lightness: -30%)
```

Comment:
The new arguments control the color when clicked/active. The defaults are based on the first color passed in (`#dfdfdf`) when we really want darker variants of the primary color.